tools/testing/e2e/test_utils.c

```diff
@@ -165,20 +194,20 @@ bool iree_test_utils_result_elements_agree(iree_test_utils_e2e_value_t expected,
     // `require_exact_results` flag is set to `false`.
     case IREE_TEST_UTILS_VALUE_TYPE_F16:
       if (actual.f16_u16 == expected.f16_u16) return true;
-      if (FLAG_require_exact_results) return false;
+      if (iree_test_utils_max_elements_to_check()) return false;
```

Comment:
Typo here? Looks like this should have been `iree_test_utils_require_exact_results()` instead.